docs/dependency-injection-liftweb-scala.adoc

```diff
@@ -8,57 +8,45 @@
 == Introduction
 
 Dependency injection is an important topic in the JVM world. With Java, we usually go with Guice
-or Spring whenever we need it. However, scala has certain advantages over Java, like functions,
-that allow basic DI without resorting to entire DI frameworks.
+or Spring whenever we need it. However, Scala provides some unique advantages that allow many of the
+features needed for dependency injection without requiring an entire framework.
 
-Scala allows the use of http://jonasboner.com/real-world-scala-dependency-injection-di/[cake pattern]
-that can provide DI by composing traits.
+Scala allows the use of http://jonasboner.com/real-world-scala-dependency-injection-di/[cake pattern], which
```

Comment:
Yeah, sounds good. Merging this paragraph with the one above to make it flow better.